docs/Contributing.asciidoc

```diff
@@ -155,10 +163,7 @@ but that is no longer possible.
 As stated in the previous section, every feature implemented in both packages
 should be backed by proper tests.
 http://perldoc.perl.org/Test/Most.html[Test::Most] is used to implement those
-tests. As usual, tests are located under the `/t/` directory. In the openQA
-package, one of the tests consists of a call to
-http://perltidy.sourceforge.net/[Perltidy] to ensure that the contributed code
-follows the most common Perl style conventions.
+tests. As usual, tests are located under the `/t/` directory.
```

Comment:
I think it should be `t/`

Reply:
Yeah, I was also wondering about the leading slash. This sentence I only moved around. I will remove the slash.